src/io/flutter/jxbrowser/EmbeddedBrowserEngine.java

```diff
@@ -36,6 +37,8 @@ public EmbeddedBrowserEngine() {
     final EngineOptions options =
       EngineOptions.newBuilder(SystemInfo.isWindows ? OFF_SCREEN : HARDWARE_ACCELERATED)
         .userDataDir(Paths.get(dataPath))
+        .passwordStore(PasswordStore.BASIC)
```

Comment:
When using `BASIC` password store, Chromium stores password without encryption and doesn't require access to the system keyring. I expect this option to help.